boards.txt

```diff
@@ -1654,6 +1654,17 @@ featheresp32.menu.DebugLevel.debug.build.code_debug=4
 featheresp32.menu.DebugLevel.verbose=Verbose
 featheresp32.menu.DebugLevel.verbose.build.code_debug=5
 
+featheresp32.menu.PartitionScheme.default=Default
+featheresp32.menu.PartitionScheme.default.build.partitions=default
+featheresp32.menu.PartitionScheme.minimal=Minimal (2MB FLASH)
```

Comment:
is this one necessary? feathers are all 4MB or above, right?

Reply:
Not necessary. I wonder if there is still value in keeping it to simulate other devices? I don't feel strongly either way.

Reply:
i would suggest to remove it :) it's kinda pointless

Reply:
I am new to this. How do I make that change?

Reply:
> I am new to this. How do I make that change?

go here: https://github.com/tpitman/arduino-esp32/tree/patch-1 and modify the file directly (no need to create a PR against that branch as it is local to your own tree). Once you modify that file on that tree it will reflect in this PR.

Reply:
Thanks. Figured it out just after posting comment.

You are very patient with noobs. Thanks.

Reply:
do not worry that build fails. cause is external problem :)

Reply:
Interesting that it passed yesterday. Did something change over night?

Reply:
> Interesting that it passed yesterday. Did something change over night?

SSL cert expiration for dl.espressif.com. https://github.com/espressif/arduino-esp32/issues/3259

Reply:
Timing everything.